ports/esp32/network_lan.c

```diff
@@ -97,14 +97,21 @@ STATIC mp_obj_t get_lan(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_ar
         return MP_OBJ_FROM_PTR(&lan_obj);
     }
 
-    enum { ARG_id, ARG_mdc, ARG_mdio, ARG_power, ARG_phy_addr, ARG_phy_type };
+    enum { ARG_id, ARG_mdc, ARG_mdio, ARG_power, ARG_phy_addr, ARG_phy_type,
+           #if ESP_IDF_VERSION_MINOR >= 4 // Clock config requires at least version 4.4
```

Comment:
I suggest to use:
```c
#if ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(4, 4, 0)
```
So that it's clear what version is needed (then you don't need the comment).

Reply:
done